deployments/goerli/usdc/relations.ts

```diff
@@ -9,5 +9,17 @@ export default {
         field: async (fxRoot) => fxRoot.stateSender()
       }
     }
+  },
+  'inbox': {
```

Comment:
The Arbitrum L1 contract we interact with is just called `Inbox`, which is very generic and doesn't give any indication that it's Arbitrum-related.

We could rename this to something more specific (e.g. `ArbitrumInbox`) if we wanted to make it more clear what this contract does.

Reply:
Yeah agreed, I think we should prepend these aliases with `arbitrum` to differentiate from all the other cross-chain contracts relations/roots will eventually have.

Reply:
Prepended `arbitrum` to all of these aliases.